demos/default/package.json

```diff
@@ -4,7 +4,9 @@
   "private": true,
   "description": "",
   "scripts": {
-    "build": "next build"
+    "build": "next build",
+    "dev": "next dev",
+    "serve": "next build && next start"
```

Comment:
[dust] would be great if we had a 'clean' command as well that removes all of the build artifacts as there's been times where some weird problems have resolved themselves after having those build artifacts removed prior to a build

Reply:
good point, though afaik there is no Next command for that? i have added a draft issue for it